docs/make.jl

```diff
@@ -1,20 +1,21 @@
-using Documenter, PkgTemplates
+using Documenter: Documenter, makedocs, deploydocs
+using PkgTemplates: PkgTemplates
 
 makedocs(;
     modules=[PkgTemplates],
-    format=Documenter.HTML(),
+    authors="Chris de Graaf, Invenia Technical Computing Corporation",
+    repo="https://github.com/invenia/PkgTemplates.jl/blob/{commit}{path}#L{line}",
+    sitename="PkgTemplates.jl",
+    format=Documenter.HTML(;
```

Comment:
```suggestion
    format=Documenter.HTML(;
            prettyurls = prettyurls = get(ENV, "CI", nothing) == "true",
```
When we have different markdawn files, this option helps to make separate html files in local build instead of folders.

Reply:
Sory I meant this for the template not for PkgTemplate documentation itself

Reply:
Oh wow, I wish I had known this option existed before! Thanks.